mmv1/products/compute/terraform.yaml

```diff
@@ -1498,6 +1513,12 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           neg_name: "my-lb-neg"
           network_name: "neg-network"
           subnetwork_name: "neg-subnetwork"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "network_endpoint_group_non_gcp"
+        primary_resource_id: "neg"
+        vars:
+          neg_name: "my-lb-neg"
+          network_name: "neg-network"
```

Comment:
It looks like `TestAccComputeNetworkEndpointGroup_networkEndpointGroupNonGcpExample` is failing with the following error:

```
Error: "name" ("") doesn't match regexp "^(?:[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?)$"
on terraform_plugin_test.tf line 17, in resource "google_compute_subnetwork" "default":
17:   name          = ""
```

I think you may need to add a `subnetwork_name` variable value here.

Reply:
Whoops great catch! In all of my forwarding rule excitement, I forgot to go back and fix this test. It actually was a little out of date - subnetworks aren't allowed with non-GCP, so these examples shouldn't have a subnetwork (I updated that for the forwarding rule, but not for this one).

Updated the `network_endpoint_group_non_gcp.tf.erb` file to remove the subnetwork, generated the provider, and ran it:

![image](https://user-images.githubusercontent.com/17228751/153921980-4d6b1805-e102-46a5-b780-d6741b7414b0.png)

Thanks again!